app/Http/Controllers/ClubEventController.php

```diff
@@ -313,8 +285,15 @@ public function edit($id)
 
         // add LDAP-ID of the event creator - only this user + Marketing + CL will be able to edit
         $revisions = json_decode($event->getSchedule->entry_revisions, true);
-        $created_by = $revisions[0]["user id"];
-        $creator_name = $revisions[0]["user name"];
+        if (is_null($revisions)) {
```

Comment:
I encountered schedules where `entry_revisions` was the empty string. Then `json_decode` will return `NULL`.